pkgs/development/python-modules/tensorboardx/default.nix

```diff
@@ -0,0 +1,28 @@
+{lib, fetchFromGitHub, buildPythonPackage, matplotlib, numpy, pillow, pytorch, protobuf, six, pytest, torchvision }:
+
+buildPythonPackage rec {
+  pname = "tensorboardx";
+  version = "1.6";
```

Comment:
Can we update this to the latest version?

Reply:
@mmahut, done. Library became even larger. I had to disable some tests. Most failed for spurious reasons, like "44100" vs "44100.0" in audioclip in serialized prototxt.